Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAi_test.py

```diff
@@ -120,3 +120,14 @@ def test_get_user_details(requests_mock):
     result = get_user_details(loginClient, queryClient, user)
     assert result.outputs_prefix == 'ConcentricAI.UserDetails'
     assert result.outputs_key_field == 'info'
+
+
+def test_get_file_sharing_details(requests_mock):
```

Comment:
Unit tests should contain given-when-then comment to describe the unit test scenario. Given <some input>, When <some state> Then <ensure some behavior>.

Reply:
@shams-concentricai Please add next to the `given-when-then` comments a short verbal description of what is given (next to **given**), and what's exactly being asserted (next to **then**).

Reply:
Done !